TF-3610 improvement/add order alphabetical

joecan opened the pull request:
##### *To:*
@inancsevinc @jjinux @kevinzang @team-f

##### *What:*
Add import-order-alphabetical/order rule. Aso add vscode to gitignore. Needed to fix the order of imports of several unrelated files.

##### *JIRA:*
https://udemyjira.atlassian.net/browse/TF-3610

##### *What did you test:*
yarn test

##### *What dashboards will you be monitoring:*
N/A

joecan committed:
improvement: add import-order-alphabetical/order rule

joecan committed:
improvement: add vscode settings dir to gitignore

jjinux commented on .gitignore:
```diff
@@ -2,3 +2,6 @@ node_modules
 .idea
 lerna-debug.log
 npm-debug.log
+
```
Don't bother with the comment. Can you sort the lines?

joecan committed:
fix: sorted and removed vscode comment

jjinux reviewed (approved):
LGTM

joecan committed:
Publish

 - babel-polyfill-udemy-website@7.0.7
 - babel-preset-udemy-website@8.0.7
 - eslint-config-udemy-basics@5.0.3
 - eslint-config-udemy-website@7.1.5
 - eslint-plugin-udemy@6.0.3
